Bump version

scott-w opened the pull request:
I've bumped the version number and added tested support for Lodash.

This bumps a fair few dependencies that I needed to make this build work. I'd appreciate a bit of a look over this to see if there's anything we need to pick up on. My feeling is we should roll a v1.0.0 release and try to keep this working alongside Mn 3.x if and when we're able to.

Related Issues #131 #132

scott-w commented:
Finally got all the deps updated so Travis can build+test. Yay for trial and error!

scott-w commented on bower.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.6.3",
+  "version": "1.0.0",
```
Feeling is that Syphon has depended on jQuery for so long now that changing it for v1 would potentially break code and assumptions.

I do agree that it would be nicer to remove the dependencies going forward. I do see a lot of code transitioning away from jQuery over time.

Perhaps we resolve to remove jQuery for v2?

scott-w replied:
That said, if that's the direction we look to go, I think I should re-add support for jQuery 2.x as it's still widely used and supported.

blikblum replied:
According to semver, breaking changes can occur before 1.0 at will.

If we release, e.g., 0.7 we have the freedom to decide upfront although since the jQuery usage is pretty basic, i believe that will have no regressions

Another thing that i'd like to propose is putting the dependencies as peerDependencies like we do for Marionette, not sure this will be configured as a breaking change

I think that jQuery 1 and 2 should be supported since there's no reason to not support them

scott-w replied:
Sounds good, I'll make those changes as soon as I get chance.

scott-w commented:
@blikblum I've updated to allow a range of jQuery deps and dropped the package version to 0.7.0. Can you check I've got the bower.json package version correct?

blikblum commented:
I don't know about bower.
CI is failing with EPeerInvalid in node v4: The package jquery@3.1.1 does not satisfy its siblings' peerDependencies requirements!

While building another library i learned that is necessary to put the peerDependencies in devDependencies to work.

blikblum commented:
LGTM. I think we can ship it

scott-w commented:
:shipit:
